julia-repl.el

```diff
@@ -211,16 +211,9 @@ Creates INFERIOR-BUFFER-NAME (‘make-term’ surrounds it with *s),
 running EXECUTABLE-PATH.
 
 Return the inferior buffer.  No setup is performed."
-  (apply #'make-term inferior-buffer-name executable-path nil
-         (julia-repl--split-switches)))
-
-(defun julia-repl--setup-captures ()
-  "Set up captured keys which are captured from ‘term’.
-
-Note that this affects ‘term’ globally."
-  (mapc (lambda (k)
-          (define-key term-raw-map k (global-key-binding k)))
-        julia-repl-captures))
+  (case julia-repl-buffer-backend
```

Comment:
```elisp
(case julia-repl-buffer-backend
  (term (do-term-stuff)
  (comint (do-comint-stuff))
```
...is a pattern I use in a few places. It might be worth at least adding some error checking (in case the value isn't 'comint or 'term) or even making a function or macro that wraps this, but I'm not sure.

Reply:
you could use either `cl-ecase` or `pcase-exhaustive` which would both error on unmatched cases.